model/project_ref.go

```diff
@@ -1564,6 +1564,16 @@ func FindOneProjectRefByRepoAndBranchWithPRTesting(owner, repo, branch, calledBy
 		})
 		return nil, nil
 	}
+	if repoRef.RemotePath == "" {
+		grip.Error(message.Fields{
+			"source":  "find project ref for PR testing",
+			"message": "repo ref has no remote path, cannot use for PR testing",
+			"owner":   owner,
+			"repo":    repo,
+			"branch":  branch,
+		})
+		return nil, errors.Wrapf(err, "repo ref '%s' has no remote path, cannot use for PR testing", repoRef.Id)
+	}
```

Comment:
Can we add a check for this case to `TestFindOneProjectRefByRepoAndBranchWithPRTesting`?

Reply:
Sure, updated one of the existing test cases to check this case